test/utils/test_utils.py

```diff
@@ -59,3 +60,53 @@ def test_validate_controls():
         validate_control_variables([arr], [arr[:1]])
     u_mod = validate_control_variables([arr], [arr], trim_last_point=True)
     assert u_mod[0].n_time == 1
+
+
+@pytest.mark.parametrize(
+    ["thresholder", "expected"], [("l0", 4), ("l1", 6), ("l2", 10)]
+)
+def test_get_regularization_1d(thresholder, expected):
+    data = np.array([[0, 1, 2]]).T
+    lam = 2
```

Comment:
Use unique prime numbers only.  This makes it more likely that the tests will accidentally pass when something's wrong.